keyboards/ph_arisu/readme.md

```diff
@@ -0,0 +1,16 @@
+# PH ARISU
+
+![PH_ARISU](https://i.imgur.com/qr2Tn2X.jpg)
+
+PCB is base on https://github.com/FateNozomi/arisu-pcb
+and used for a local groupbuy in the Philippines.
+
+* Keyboard Maintainer: [Neil Brian Ramirez](https://github.com/NightlyBoards)
```

Comment:
Did NightlyBoards do the PCB modification (from the FateNozomi Arisu)? Usually, this line would refer to your account.

Reply:
Yes is the one modified the PCB I just do the QMK and VIA for him.